### PR TITLE
update dev guide

### PR DESCRIPTION
**Pull request by jesusholoo**
Update and fix some lines of the dev guide

### CODE REVIEW

File: docs/developers/dev-guide.md [hunk 1]
```diff
@@ -117,9 +114,11 @@ There are some rules on how coordinators must process transactions. These rules
...
+- **L1L2-batch**: Both L1 and L2 transactions can be mined. In these batches, the coordinator must forge the last L1 queue.
+
+In both cases the coordinator may include L1-coordinator-transactions.
 
-Apart from the fact that the coordinator is only able to claim fees for those transactions that are correctly processed, a coordinator is required to process L1 user transactions periodically.  The smart contract establishes a deadline for the L1L2-batches. This deadline indicates the maximum time between two consecutive L1L2 batches. Once this deadline is reached, the coordinator cannot forge any L2 transactions until the deadline is reset which only happens after a L1L2 batch is forged.
+The coordinators are obliged to process L1 user transactions periodically. The smart contract establishes a deadline for the L1L2-batches. This deadline indicates the maximum time between two consecutive L1L2 batches. Once this deadline is reached, the coordinator cannot forge any L2 batches until the deadline is reset which only happens after a L1L2 batch is forged.
```
Comment (druiz0992) [inline docs/developers/dev-guide.md]: Coordinators must process L1 user transactions periodically.

**Review by druiz0992 — CHANGES_REQUESTED** [review]
Thanks for the review

File: docs/developers/dev-guide.md (hunk 1 above)
Comment (jesusholoo) [reply docs/developers/dev-guide.md]: fixed!

**Review by druiz0992 — DISMISSED** [review]

File: docs/developers/dev-guide.md [hunk 2]
```diff
@@ -158,14 +157,15 @@ The WithdrawalDelayer smart contract can be in one of two states:
 1. **Normal Mode**: Amount above withdrawal limit is available for withdrawal, but with a delay D. This is the standard state.
 2. **Emergency Mode**: The Hermez Foundation is the only body that may change the 
 WithdrawalDelayer mode to Emergency in case of an attack. In this scenario, funds can only be 
-withdrawn by the governance under the tutelage of Aragon court. 
+withdrawn by the governance under the tutelage of a emergency council that will return the funds to the users.
```
Comment (krlosMata) [inline docs/developers/dev-guide.md]:
```suggestion
withdrawn by the governance under the tutelage of an emergency council that will return the funds to the users. 
```

File: docs/developers/dev-guide.md [hunk 3]
```diff
@@ -158,14 +157,15 @@ The WithdrawalDelayer smart contract can be in one of two states:
 1. **Normal Mode**: Amount above withdrawal limit is available for withdrawal, but with a delay D. This is the standard state.
 2. **Emergency Mode**: The Hermez Foundation is the only body that may change the 
 WithdrawalDelayer mode to Emergency in case of an attack. In this scenario, funds can only be 
-withdrawn by the governance under the tutelage of Aragon court. 
+withdrawn by the governance under the tutelage of a emergency council that will return the funds to the users. 
 
 ## Adding New Tokens
 Hermez contains a list with all tokens supported. The following list includes some requirements on the token listing:
 - Tokens must be ERC20
 - Only the governance can add new tokens.
```
Comment (krlosMata) [inline docs/developers/dev-guide.md]: This has been changed. Anyone can add tokens.

**Review by krlosMata — CHANGES_REQUESTED** [review]

**Review by krlosMata — APPROVED** [review]
:smile_cat:

**Review by druiz0992 — APPROVED** [review]

File: docs/developers/dev-guide.md [hunk 4]
```diff
@@ -158,14 +157,14 @@ The WithdrawalDelayer smart contract can be in one of two states:
...
-withdrawn by the governance under the tutelage of Aragon court. 
+withdrawn by the governance under the tutelage of an emergency council that will return the funds to the users. 
 
 ## Adding New Tokens
 Hermez contains a list with all tokens supported. The following list includes some requirements on the token listing:
 - Tokens must be ERC20
-- Only the governance can add new tokens.
 - There can be up to 2^{32} different tokens.
 - Contracts maintain a list of all tokens registered in the rollup and each token needs to be listed before using it.
-- There cannot be two tokens with the same ethereum address.
```
Comment (jleebee) [inline docs/developers/dev-guide.md:169]: Ethereum

**Review by jleebee — CHANGES_REQUESTED** [review]
Change to a capital "E" for "ethereum"

**Review by jleebee — APPROVED** [review]

**Pull request merged**